doc/ale-go.txt

```diff
@@ -219,6 +219,25 @@ g:ale_go_govet_options                                 *g:ale_go_govet_options*
   This variable can be set to pass additional options to the go vet linter.
 
 
+===============================================================================
+revive                                                          *ale-go-revive*
+
+g:ale_go_reivve_executable                         *g:ale_go_revive_executable*
```

Comment:
`g:ale_go_reivve_executable` -> `g:ale_go_revive_executable`

Reply:
@liaoishere Just found a small typo when re-reading the PR

Reply:
nice catch, I’ll fix it